test/Microsoft.Azure.WebJobs.Host.EndToEndTests/DynamicConcurrencyEndToEndTests.cs

```diff
@@ -666,12 +667,13 @@ public void OnTimer(object state)
                                 // Demonstrates how a listener integrates with Dynamic Concurrency querying
                                 // ConcurrencyManager and limiting the amount of new invocations started.
                                 var concurrencyStatus = _concurrencyManager.GetStatus(_descriptor.Id);
-                                currentBatchSize = Math.Min(concurrencyStatus.AvailableInvocationCount, 32);
+                                int availableInvocationCount = concurrencyStatus.GetAvailableInvocationCount(_outstandingInvocationCount);
+                                currentBatchSize = Math.Min(availableInvocationCount, 32);
                                 if (currentBatchSize == 0)
                                 {
                                     // if we're not healthy or we're at our limit, we'll wait
                                     // a bit before checking again
-                                    _timer.Change((int)concurrencyStatus.NextStatusDelay.TotalMilliseconds, Timeout.Infinite);
+                                    _timer.Change(1000, Timeout.Infinite);
```

Comment:
Just double-checking that this change was intentional?

Reply:
Yes - I've removed ConcurrencyStatus.NextStatusDelay. Listeners can decide themselves the delay to use before checking again when no invocations are currently available.